tests/snuba/search/test_backend.py

```diff
@@ -404,6 +404,14 @@ def test_status(self):
             "status:[resolved, muted]", [self.group2, group_3], [self.group1]
         )
 
+    def test_substatus(self):
+        with Feature("organizations:issue-states"):
+            results = self.make_query(search_filter_query="is:ongoing")
+            assert set(results) == {self.group1}
+
+        with pytest.raises(InvalidSearchQuery):
```

Comment:
```suggestion
        with pytest.raises(InvalidSearchQuery, match="The substatus filter is not supported for this organization"):
```

And/or, I'd add a comment about how now the feature is off and that's why we're expecting an error.